src/envoy/transcoding/config.cc

```diff
@@ -134,10 +134,6 @@ Config::Config(const Json::Object& config) {
         default:
           break;
       }
-
-      //      pmb.Register("POST", "/" + service->full_name() + "/" +
```

Comment:
So method->options().GetExentsion() is always there even you did not specify the "options" in the proto file.  Is that right?